### DISCUSSION TITLE
Responsive margins

### DISCUSSION THREAD

**Discussion by ChrisT44** [twbs/bootstrap#38607] [Q&A] [answered]
I'm sure I used to be able to do this and I don't understand why it would be taken away. I want to apply a margin on the x-axis but only when the screen is large enough for it to look right. I was trying to use; "mx-xl-5" - but this results in no margin. I'm sure this was possibly before. The documentation states:

> Spacing utilities that apply to all breakpoints, from xs to xl, have no breakpoint abbreviation in them. This is because those classes are applied from min-width: 0 and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation. The classes are named using the format {property}{sides}-{size} for xs and {property}{sides}-{breakpoint}-{size} for sm, md, lg, and xl.

...I don't understand this documentation. Which other breakpoints are there? Sorry, I'm probably being stupid but can somebody point out what I'm doing wrong?

**Comment by julien-deramond** [comment, marked as answer]
If we are taking into account the `5` size (`3rem` for `margin-right` and `margin-left`)  you can use the following breakpoints for an `mx` approach:
* `.mx-sm-5`
* `.mx-md-5`
* `.mx-lg-5`
* `.mx-xl-5`
* `.mx-xxl-5`

You can find them defined at https://github.com/twbs/bootstrap/blob/main/dist/css/bootstrap.css

**Comment by ChrisT44** [comment]
This was just me being stupid. The page I was using the class on was loaded within an iframe and thus the "screen size" didn't match the media query for mx-xl-* 

Sorry for being so stupid and thanks for your reply!
